compiler-core/src/language_server/engine.rs

```diff
@@ -246,7 +259,17 @@ where
                 return Ok(None);
             };
 
+            let type_qualifiers = get_type_qualifiers(module);
+            let module_qualifiers = get_module_qualifiers(module);
```

Comment:
These are only used in the code action function so let's move these function calls in there.

Reply:
Got it.